modules/perception/camera/app/cipv_camera.cc

```diff
@@ -864,21 +865,29 @@ bool Cipv::CollectDrops(const base::MotionBufferPtr &motion_buffer,
 
     object_id_skip_count_[cur_id] = 0;
 
+    auto pos = world2camera * obj->center;
+    float center_x = static_cast<float>(pos(2));
+    float center_y = static_cast<float>(-pos(0));
+
     object_trackjectories_[cur_id].push_back(
-        std::make_pair(obj->center(0), obj->center(1)));
+        std::make_pair(center_x, center_y));
```

Comment:
`emplace_back(center_x, center_y);` is shorter.

Reply:
Thank you for the review and suggestion. Changed!